README.md

```diff
@@ -130,6 +131,11 @@ They also serve as examples to help build custom probes yourself.
 | vxlan_status | Detect devices with VXLAN interface that is not up |
 | static_vxlan_vtep_anomalies | Verify VXLAN intent by ensuring all L2 segments have expected VTEP flood list |
 
+## Security
+
+| Probe | Description
+| ---------- | -----------
+| acl_stat_anomalies | Report on acl rule matches that exceed user defined thresholds |
 
 # Getting Started
```

Comment:
We need to remove/rephrase this section and anything else that talks about jupyter playbooks.

Reply:
Updated and also removed unused images.